VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -271,12 +271,18 @@ def try_parse_context(self):
             self._change_log_file()
             self._context._status_dir = handler_env['handlerEnvironment']['statusFolder']
             self._context._heartbeat_file = handler_env['handlerEnvironment']['heartbeatFile']
-            self._context._seq_no = self._get_current_seq_no(self._context._config_dir)
+            if seqNo != -1:
+                self._context._seq_no = seqNo
+            else:
+                self._context._seq_no = self._get_current_seq_no(self._context._config_dir)
             if self._context._seq_no < 0:
                 self.error("Unable to locate a .settings file!")
                 return None
             self._context._seq_no = str(self._context._seq_no)
-            self.log('sequence number is ' + self._context._seq_no)
+            if seqNo != -1:
+                self.log('sequence number from environment varaible is ' + self._context._seq_no)
+            else:
+                self.log('sequence number is ' + self._context._seq_no)
```

Comment:
change this as well to 'sequence number based on config file-names is '

Reply:
done